spec/mastermind_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "mastermind"
+
+RSpec.describe Mastermind do
+  describe "#mastermind" do
+    let(:output) { StringIO.new }
+
+    it "won all guesses" do
+      guesses = ["RED", "GREEN", "BLUE", "YELLOW"]
+      input = String.new(guesses.join("\n"))
```

Comment:
There's no need to do `String.new(...)`, a simple `guesses.join("\n")` is sufficient

Reply:
Noted